### DISCUSSION TITLE
Edited site.css not loading

### DISCUSSION THREAD

**Discussion by CybrZr00** [Dotnet-Boxed/Templates#175] [Q&A]
﻿**Which project template is the issue about?**
Asp.Net Core MVC 6 Boilerplate
...

**Which version of the project template is the issue about?**
current - unsure - added to VS today
...

**Which version of Visual Studio or the dotnet SDK were you using?**
Visual Studio 2017
...

As a novice I wanted to edit this template, I have added EntityFramework and identity etc.  Whilst styling I have come across an issue which I'm stuck on.  After changing the body section of site.css to have a background-image, the page loaded with it just fine, but after editing the Home/index view however, the image didn't show and the app loaded a version of the css file where there are no changes.  I have tried other browsers, clearing the caches and cookies etc, closing the browser, restarting Visual Studio, and can't for the life of me get the changed version of the css file to load.

I have also noticed there is an error in the site.js showing in Chromes console which may or may not be related:

> Uncaught ReferenceError: exports is not defined

Which is some sort of calculator function.

Hope this is enough for you to go on...
I used IIS,
No cloud provider,
No TypeScript etc,
.net core

**Comment by CybrZr00** [comment]
Weirdly enough, it seems to be working again.  The changes had been removed, so most likely user error. However the site.js error relating to exports is still present.

Edit: Should be using styles.css :)

**Comment by RehanSaeed** [comment]
`gulp` is used to create the `site.css` file each time you update any `.scss` file. Sometimes `gulp` is a bit flaky and does not work correctly or stops working if your `scss` is invalid. Glad it sorted itself out.
